gradle/libs.versions.toml

```diff
@@ -2,15 +2,15 @@
 androidGradle = "8.1.0"
 androidxActivity = "1.5.0"
 androidxAppCompat = "1.3.1"
-androidxCompose = "1.5.3"
-androidxComposeCompiler = "1.5.3"
+androidxCompose = "1.6.1"
+androidxComposeCompiler = "1.5.9"
```

Comment:
Do we really need to update these? Updating them means whoever is using this library, has to update Compose, since we don't change anything here might as well keep the old versions

Reply:
We need to update compose, because of the kotlin version bump:

```
This version (1.5.3) of the Compose Compiler requires Kotlin version 1.9.10 but you appear to be using Kotlin version 1.9.22 which is not known to be compatible.  Please consult the Compose-Kotlin compatibility map located at https://developer.android.com/jetpack/androidx/releases/compose-kotlin to choose a compatible version pair (or `suppressKotlinVersionCompatibilityCheck` but don't say I didn't warn you!).
```